### ISSUE TITLE
Add actual parent post to comment pages

### ISSUE THREAD

**Issue by Subroutine7901** [jerimiah-smiggins/smiggins#282]
On the comment pages, there's a link "Go to original post". I think it would be more convenient if you could see the parent post and go to it almost like the replies, where it displays the whole parent post. If needed, it could be under a drop-down or something, kinda like the content warnings.

**Comment by trinkey** [comment]
reply backfilling aughghh

**Comment by Subroutine7901** [comment]
Not like all of the parent posts, just a preview... This isn't Sharkey...

**Comment by trinkey** [comment]
yeah but if i did that i should set up actual relationships in the database instead of just an id

**Comment by Subroutine7901** [comment]
True.... but we love optimizing the database, don't we?
Do we not?
